test/functional/eval/null_spec.lua

```diff
@@ -60,7 +60,7 @@ describe('NULL', function()
     -- FIXME should be accepted by writefile(), return {0, {}}
     null_expr_test('is accepted as an empty list by writefile()',
                         ('[writefile(L, "%s"), readfile("%s")]'):format(tmpfname, tmpfname),
-                        'E484: Can\'t open file ' .. tmpfname, {0, {}})
+                        'E484: Cannot open file ' .. tmpfname, {0, {}})
```

Comment:
? I thought locale is already being set for tests somewhere. Though I am running `en_US.UTF-8` in case it is not.